### PR TITLE
Implement build-install (as per #79)

### LINKED ISSUE #79: Simplify installation process
Hi,

I had a thought: the most common use case for kerl is to install various versions of Erlang, why not make that really simple to do by removing some of the 'extra' steps?  For example:

```
kerl build-install git https://github.com/erlang/otp.git OTP-17.1.2 17.1.2
```

Builds and installs in .kerl/installations/17.1.2/

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
~I'm still missing the bash and zsh completions, for which I have limited experience, but hope to do them in the scope of this pull request, when time allows.~

Closes #79.

TODO:

- [x] shell completion update for `zsh`
- [x] shell completion update for `bash`
- [x] tests for ~CircleCI~ GitHub Actions
- [x] declare variables with more descriptive names (`third` and `fifth` is no good)

### CODE REVIEW

**Comment by garazdawi** [comment]
It would also be good to add tests for this feature.

**Comment by paulo-ferraz-oliveira** [comment]
> would also be good to add tests for this feature.

I'll add that to the TODO list, thanks.

File: kerl [hunk 1]
```diff
@@ -2323,6 +2356,31 @@ case "$1" in
             fi
         fi
         ;;
+    build-install)
+        third="$3"
```
Comment (jadeallenx) [inline kerl:2360]: can you please use more descriptive names here like `install_dir_or_x` or `build_name_or_y` where x and y are descriptive

Comment (paulo-ferraz-oliveira) [reply kerl:2360]: Sure thing... I'll add that to the list of actions in the description.

Comment (paulo-ferraz-oliveira) [reply kerl]: Pushed.

**Comment by paulo-ferraz-oliveira** [comment]
Force-pushed to rebase onto `master` (in order to include GitHub Actions -based tests).

**Comment by paulo-ferraz-oliveira** [comment]
I've no idea why this is breaking in Actions. Locally it works, but I may have some difference in state (?). @jadeallenx, would you be able to help me here? Thank you.

**Review by mervyd — APPROVED** [review]

**Comment by jadeallenx** [comment]
Sorry I've lost the thread on this - is the CI busted still?

**Comment by paulo-ferraz-oliveira** [comment]
Hey. It is yeah, sorry. Haven't gotten around to this.

**Comment by paulo-ferraz-oliveira** [comment]
Trying to get this into shape again:
* 94aa6bb - update CI to remove a warning
* d81362e - act on CI results to fix the feature implementation

**Comment by jadeallenx** [comment]
C'mon CI hamsters 🐹

**Comment by paulo-ferraz-oliveira** [comment]
I'm closer to finishing. Have to look closer at how `path` works, since that's where it's failing now.

**Comment by paulo-ferraz-oliveira** [comment]
Hurrah, [it's working](https://github.com/kerl/kerl/actions/runs/4920587822/jobs/8789533909#step:15:1). I'll still prefer to rebase this on top of https://github.com/kerl/kerl/pull/444, once that gets merged to `master`, for "All ✅".

Moving to **Ready for review**...

**Comment by jadeallenx** [comment]
fantastic thanks - i know it's taken a long time to get it merged

**Comment by paulo-ferraz-oliveira** [comment]
Thanks much. We now have ✅ in `master` so everything's good 😄

**Pull request merged**